.vscode/launch.json

```diff
@@ -6,10 +6,10 @@
       "type": "node",
       "request": "launch",
       "runtimeExecutable": "yarn",
-      "args": ["jest", "--runInBand"],
+      "args": ["jest", "--runInBand", "--coverage=false"],
```

Comment:
no coverage?

Reply:
when debugging, yes. Without that, every time you step into a function, it steps into a bunch of jest-coverage-wrapping functions first.